src/apify_helpers.js

```diff
@@ -213,6 +213,168 @@ const getPrefilledValuesFromInputSchema = (inputSchemaStringJSON) => {
     return prefilledObject;
 };
 
+/**
+ * Converts Apify input schema to Zapier input fields.
+ * Input schema spec.
+ * https://docs.apify.com/platform/actors/development/actor-definition/input-schema Fields schema
+ * spec. https://zapier.github.io/zapier-platform-schema/build/schema.html#fieldschema
+ * @param inputSchema
+ * @param actor
+ */
+const createFieldsFromInputSchemaV1 = (inputSchema, actor) => {
```

Comment:
I tried to do as much comments as it necessary, if you didn't get some part we can discuss. It was quite code heavy to map two schemas each other.